src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -497,6 +497,9 @@ writeDocx opts doc@(Pandoc meta _) = do
                        _                  -> []
 
   let docPropsPath = "docProps/core.xml"
+  let extraCorePropsMap = M.fromList 
+          [("subtitle","dc:subject"),("abstract","dc:description")
+          ,("lang","dc:language"),("category","cp:category")]
```

Comment:
Better use two lists and zip them for the Map constructor, so that one of the lists can be reused in the filter below (~515) to avoid hard-coded duplicates of pandoc metadata names.